src/Entity/MembershipType.php

```diff
@@ -67,6 +67,13 @@ class MembershipType extends ConfigEntityBase implements MembershipTypeInterface
   protected $workflow;
 
   /**
+   * The processor plugin string ID.
+   *
+   * @var string
+   */
+  protected $processor_plugin_id;
```

Comment:
This was/is an instance of \Drupal\membership\Plugin\MembershipProviderInterface. I was thinking it would be useful to have some membership types control the plugin for their membership instances.

However, after getting something actually working, I reached the conclusion you were correct, it needs to be on the Membership instance, not on MembershipType. Will remove.